src/components/lobby/LobbyPage.js

```diff
@@ -162,9 +174,21 @@ export default function LobbyPage() {
             />
           </form>
         )}
-        <Typography variant="h5" className={classes.margin}>
-          Game starts when everyone is ready
-        </Typography>
+        <Typography>{`Total number of custom words: ${numOfCustomWords}`}</Typography>
```

Comment:
i just fixed this and that up but accidentally committed instead of commented but by wrapping in a brackets you can use some jsx to add a space between the colon and the numOfCustomWords

Reply:
Awesome!